Sources/ATProtoKit/Networking/SessionManager/ATProtocolConfiguration.swift

```diff
@@ -12,10 +12,10 @@ import Logging
 public class ATProtocolConfiguration: ProtocolConfiguration {
 
     /// The user's handle identifier in their account.
-    public private(set) var handle: String
+    public var handle: String
 
     /// The app password of the user's account.
-    public private(set) var appPassword: String
+    public var appPassword: String
 
     /// The URL of the Personal Data Server (PDS).
     public private(set) var pdsURL: String
```

Comment:
```swift
public var pdsURL: String
```

I'd also like `pdsURL` to have `private(set)` removed. Do you mind adding that as well? Otherwise, once you do that, I'll approve it and put it in the next available hotfix.

Reply:
Nice catch, just committed the change

Reply:
Looks like GitHub messed up the auto commit. Let me fix the spacing

Reply:
Should be good now